tinkerbell/rufio/templates/deployment.yaml

```diff
@@ -32,7 +32,9 @@ spec:
         command:
         - /manager
         args:
-        - --leader-elect
+        {{- range .Values.args }}
+        - {{ . }}
+        {{- end }}
```

Comment:
this might need to hardcode `--leader-elect` and then additional args can be added. In its current state if the use modifies the rufio args from the stack they'll need to be aware that they will also need to supply `--leader-elect`. Smee handles this with an `additionalArgs` https://github.com/tinkerbell/charts/blob/8ca2213ea20d33a59caafab1d6425add022d1cfe/tinkerbell/smee/values.yaml#L119